tests/test_transport_xhrstreaming.py

```diff
@@ -32,7 +30,7 @@ async def test_process_OPTIONS(make_transport):
     assert resp.status == 204
 
 
-async def test_session_has_request(make_transport, make_fut):
-    transp = make_transport(method="POST")
-    transp.session._remote_messages = make_fut(1)
-    assert isinstance(transp.session.request, web.Request)
+# async def test_session_has_request(make_transport, make_fut):
```

Comment:
I think that it is better to remove commented out code, take a look at `flake8-eradicate`